packages/core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hds-core",
-  "version": "3.0.0-alpha.0",
```

Comment:
Aren't these version changes here a bit weird since they are already in `development`. Did you rebase?

Reply:
Seeing extra commit https://github.com/City-of-Helsinki/helsinki-design-system/pull/1139/commits/2377a5a2d537c36457469039f6459d11c284800f that should be dropped

Reply:
Rebased to most current development and that fixed the issue of extra commit appearing on this PR.